pkg/kubelet/apis/device-plugin/v1alpha1/constants.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deviceplugin
+
+const (
+	// Healthy means that the device is healty
+	Healty = "Healthy"
+	// UnHealthy means that the device is unhealty
+	Unhealthy = "Unhealthy"
+
+	// HeartbeatOk means that the heartbeat succeeded
+	HeartbeatOk = "Heartbeat ok"
+	// HeartbeatOk means that the heartbeat did not succeed
```

Comment:
That makes a lot of sense!
I actually don't think an error should ever be returned and that in case of failure the device plugin should always register himself